storages/hdf5/dh5lite.c

```diff
@@ -41,22 +41,22 @@ typedef struct {
 
 
 /* Error macros for when DLiteDataModel instance d is available */
-#define DFAIL0(d, msg) \
-  do {err(-1, "%s/%s: " msg, d->s->location, d->uuid); goto fail;} while (0)
+#define DFAIL0(code, d, msg) \
```

Comment:
This is old code. It should preferable include dlite-errors.h rather than redefining these macros locally. But it can be done in a separate issue.

Reply:
I just moved it into dlite-macros.h for now ...